### PR TITLE
ISLE-v.1.2.0 PR for release

### PR DESCRIPTION
**Pull request by g7morris**
@bseeger When you have a chance, please review this ISLE-v.1.2.0 PR for release. Thank you.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM islandoracollabgroup/isle-tomcat:1.1.2
+FROM borndigital/isle-tomcat:1.2.0-dev
```
Comment (bseeger) [inline Dockerfile]: This needs to be updated to islandoracollabgroup/isle-tomcat:1.2.0

File: README.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
 Based on:  
  - [ISLE-Tomcat](https://github.com/Islandora-Collaboration-Group/isle-tomcat)
- - Fedora Repository 3.8.1
- - @todo: complete me
 
-# Generic Usage
+Contains and Includes:
+ - [Fedora Repository 3.8.1](https://duraspace.org/fedora/)
+ - [Gsearch 2.9](https://github.com/discoverygarden/gsearch.git) (_DGI fork_)
+ - [Maven 3.6.0](https://maven.apache.org/)
```
Comment (bseeger) [inline README.md]: This needs to be changed to 3.6.1
Ant needs to be changed to 1.10.6
I would also be fine with leaving maven and ant out of this list.

**Review by bseeger — CHANGES_REQUESTED** [review]
Looks good overall. A few things to tweak.

File: README.md (hunk 2 above)
Comment (g7morris) [reply README.md]: I think we should leave on the list as a pre-caution given their importance in not only building the Dockerfile but they can be used by ISLE users for building other configurations once the image is in use. Those tools are installed.

File: Dockerfile (hunk 1 above)
Comment (g7morris) [reply Dockerfile]: Okay I believe that is now fixed. Apologies for missing that in first round.

**Pull request merged**